### ISSUE TITLE
Remove "FollowsFrom" relationships

### ISSUE THREAD

**Issue by iredelmeier** [open-telemetry/opentelemetry-go#193]
Reference is not defined in the specs. What does a follow from mean?

_Originally posted by @bogdandrutu in https://github.com/open-telemetry/opentelemetry-go/pull/190_

Note that this would affect FollowsFrom-based relationships in the OpenTracing bridge.

**Comment by jmacd** [comment]
"Reference" may be a misleading term. I thought we called these relationships.

I think it will be a set-back if we remove FollowsFrom.

**Comment by iredelmeier** [comment]
@jmacd that's my understanding too re: "relationships" instead of "references". The first line is a quote since I created this issue from a discussion in #190.

**Comment by rghetia** [comment]
Use attribute "span.relation" = "childOf" or "followsFrom" in Link.

**Comment by lizthegrey** [comment]
I'm confused though, doesn't `Link` lack the `relation` or `RelationshipType` field?

Is

```
Relation{
	SpanContext:      sc,
	RelationshipType: FollowsFromRelationship,
}
```

being replaced with:

```
Link {
	SpanContext: sc,
	Attributes []core.KeyValue
}
```
which lacks the `RelationshipType` field?

**Comment by lizthegrey** [comment]
and how does this relate to https://github.com/open-telemetry/opentelemetry-go/pull/225

**Comment by jmacd** [comment]
I feel we still have no substitute for "follows from", see here https://github.com/open-telemetry/opentelemetry-specification/pull/337#discussion_r341597015

**Comment by lizthegrey** [comment]
I chose to do https://github.com/open-telemetry/opentelemetry-go/pull/272 in order to implement Links at span creation time, rather than trying to munge FollowsFrom into a Link etc

**Comment by rghetia** [comment]
fixed by #451
